### ISSUE TITLE
noqa not correctly recognized for multiline variable definition

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#773]
In GitLab by @spreiter on Oct 1, 2020, 05:18

*Please describe how you installed Flake8*

Flake8 is cached by pre-commit, using the flake8-docstrings plugin. From this cache folder, I have multiple versions available, with and without the docstrings pluging and different versions. (only for testing this bug)

For comparison I installed pycodestyle (version 2.6.0), which runs without problems.

*Please provide the exact, unmodified output of `flake8 --bug-report`*
```
{
  "dependencies": [],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.10",
    "system": "Linux"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.6.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.2.0"
    }
  ],
  "version": "3.8.3"
}
```

*Please describe the problem or feature*

I use pre-commit with autopep8 *and* flake8. I have such arrays in my code, which can not be calculated, but I would rather have the in a proper structure, than everything fully PEP8 conform.

The noqa flag in the first line should quiet all following lines, which belog to the same command. This works for autopep8 *and* when running pycodestyle manually. But flake8 reports those as error E128.

I tried this with alternative cached version of pre-commit:
- 3.7.9 (mccabe: 0.6.1, pycodestyle: 2.5.0, pyflakes: 2.1.1) CPython 2.7.5 on Linux
- 3.8.3 (mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.6.10 on Linux
- 3.8.3 (flake8-docstrings: 1.5.0, pydocstyle: 3.0.0, mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 2.7.5 on Linux
- 3.8.3 (flake8-docstrings: 1.5.0, pydocstyle: 5.1.1, mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.6.10 on Linux

-> only flake8 3.7.9 does not report these errors

*Workaround:*

Add the file to the "per-file-ignores" list, but this is not, what I want actually.

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

The following example can be used to reproduce the bug:

```
mapping = [  3,   2,   1,  11,  10,   9,   8,  15,  # noqa: E12,E2
           105, 109, 106,  77,  81,  85,  89,  93,
           119, 112, 118,  64,  68,  72,  76,  80,
            84,  88,  92, 122, 124, 123, 191, 187,
           183, 179, 175, 134, 129, 133, 130, 132,
           131, 190, 186, 162, 143, 136, 142, 137,
           244, 251, 250, 252]
```

Tests done with `--isolated` flag for verification.
Also tried without error code after the `noqa` flag.

**Comment by asottile** [comment]
In GitLab by @asottile on Oct 1, 2020, 07:43

noqa only affects the line the comment is on

in your case there are errors on each of the lines but you've only excluded it from the first line

suggestion:
- ignore E12/E2 everywhere
- per-file-ignore it on that file

**Comment by asottile** [comment]
In GitLab by @spreiter on Oct 1, 2020, 08:14

Ok, so the behavior is different from pycodestyle. Too bad :(

**Comment by asottile** [comment]
In GitLab by @asottile on Oct 1, 2020, 08:19

yeah noqa was fixed in 3.8.x to fix pycodestyle's inconsistent implementation

for example this is allowed in pycodestyle:

```
mapping = [  3,   2,   1,  11,  10,   9,   8,  15,  # noqa: WAT123
           105, 109, 106,  77,  81,  85,  89,  93,
           119, 112, 118,  64,  68,  72,  76,  80,
            84,  88,  92, 122, 124, 123, 191, 187,
           183, 179, 175, 134, 129, 133, 130, 132,
           131, 190, 186, 162, 143, 136, 142, 137,
           244, 251, 250, 252]
```

**Comment by asottile** [comment]
In GitLab by @spreiter on Oct 1, 2020, 08:41

I see. Thanks for the update.
